src/Raven.Server/Documents/TcpHandlers/TcpConnectionOptions.cs

```diff
@@ -67,6 +68,10 @@ public void Dispose()
             if (_isDisposed)
                 return;
 
+#if !RELEASE
```

Comment:
Do we actually set this, I thought we had just `DEBUG`

Reply:
I copied this from HttpCache: https://github.com/ravendb/ravendb/blob/6af136312be49c596af32d3d175d289477bd50a1/src/Raven.Client/Http/HttpCache.cs#L105

DEBUG and RELEASE AFAIK are built-in?